articles/aws-api-setup.md

```diff
@@ -19,7 +19,7 @@ This is a one time setup
   ![](/media/articles/aws-api-setup/aws-api-setup-2.png)
 
 <% if (account.userName) { %>
-3. Download the metadata from <a href="https://@@account.namespace@@/samlp/metadata/@@account.clientId@@" target="_new">https://@@account.namespace@@/samlp/metadata/@@account.clientId@@</a> and upload it here.
+3. Download the metadata from <a href="https://@@account.namespace@@/samlp/metadata/@@account.clientId@@">https://@@account.namespace@@/samlp/metadata/@@account.clientId@@</a> and upload it here.
```

Comment:
I prefer to let users decide where they want to open links and think generally it is considered bad practice to use target=new. It is better to let the user decide. Additionally, it was applied highly inconsistently in docs. Many places don't have it, particularly since most links are written in markdown. 

Later I am going to go and convert all links that are in <a> format to markdown when possible.